Extension/lib/filter/antibanner.js

```diff
@@ -1016,6 +1016,23 @@ AntiBannerService.prototype = {
         return dfd;
     },
 
+    _getRulesCount: function () {

```

Comment:
This method is ineffective, it would be much better to use rules counter that is inside requestFilter